querybook/webapp/lib/sql-helper/sql-autocompleter.ts

```diff
@@ -247,17 +271,19 @@ export class SqlAutoCompleter {
         return new Promise(async (resolve) => {
             let results: ICompletionRow[] = [];
             if (lineAnalysis.context === 'table') {
-                results = (await this.getTableNamesFromPrefix(searchStr)).map(
-                    formatter.bind(null, lineAnalysis.context)
-                );
+                results = (await this.getTableNamesFromPrefix(searchStr))
+                    .map(this.hierarchicalQuotationFormatter)
+                    .map(formatter.bind(null, lineAnalysis.context));
             } else if (
                 lineAnalysis.context === 'column' &&
                 lineAnalysis.reference
             ) {
                 results = this.getColumnsFromPrefix(
                     searchStr,
                     lineAnalysis.reference
-                ).map(formatter.bind(null, lineAnalysis.context));
+                )
+                    .map(this.flatQuotationFormatter)
```

Comment:
how about just move this `.map(this.flatQuotationFormatter)` into `getColumnsFromPrefix`?

Reply:
i moved all of the quotation formatting logic to the formatter level so it would not interfere with display